kobo/settings/base.py

```diff
@@ -786,8 +786,14 @@ def __init__(self, *args, **kwargs):
 
 MONGO_DB_MAX_TIME_MS = CELERY_TASK_TIME_LIMIT * 1000
 
-SESSION_ENGINE = "redis_sessions.session"
-SESSION_REDIS = env.cache_url("REDIS_SESSION_URL", default="redis://redis_cache:6380/2")
+SESSION_ENGINE = 'redis_sessions.session'
+django_environ_redis_dict = env.cache_url(
+    'REDIS_SESSION_URL', default='redis://redis_cache:6380/2'
+)
+django_environ_redis_dict['url'] = django_environ_redis_dict['LOCATION']
```

Comment:
This retains the LOCATION and ENGINE in the dict which aren't used by SESSION_REDIS. If for some reason there was ever a collision of values, it could introduce expected errors. I'd prefer to make a new dict instead of merging.